app/src/main/java/com/endava/myendava/activities/SignInActivity.java

```diff
@@ -115,14 +119,17 @@ private void handleForgotPassword() {
     private void validateEmail() {
         String email = mUserEmailEditText.getText().toString();
         if (email.length() > 0) {
```

Comment:
Why check if the length is > 0? if the input is an empty String the server will return a false value anyway.